.github/scripts/setup_runners.sh

```diff
@@ -68,7 +68,7 @@ fi
 SETUP_ACTION=$1
 SETUP_INSTANCE=`echo $2 | sed 's/,/ /g'`
 SETUP_TOKEN=$3
-SETUP_AMI_ID="ami-042f29d2ac35db697"
+SETUP_AMI_ID="ami-0d4131ed63328e32f"
```

Comment:
Can you mention does this AMI contains? Is it a plain centos/amazonlinux box or anything prerequisites required to consider it as self hosted runner?

Reply:
The AMI remove the staled 2.272 version of old runner binary. That is the only change since we are dynamically retrieve the latest runner binary now in the script.

Reply:
so it is a plain amazon linux machine now?

Reply:
nope, it has necessary packages to run all the integTest and all the support packages for scripts.